### PR TITLE
s3cmd: add page

### PR DESCRIPTION
**Pull request by bgbak**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):* 2.2.0*

### CODE REVIEW

File: pages/common/s3cmd.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Get file from Bucket:
+
+`s3cmd get s3://{{bucket}}/{{file}} {{local_file}}`
+
+- Put file into bucket:
+
+`s3cmd put {{local_file}} s3://{{bucket}}/{{file}}`
+
+- Move object:
```
Comment (navarroaxel) [inline pages/common/s3cmd.md:26]:
```suggestion
- Move an object to a specific bucket location:
```

File: pages/common/s3cmd.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`s3cmd mv s3://{{src_bucket}}/{{src_object}} s3://{{dst_bucket}}/{{dst_object}}`
+
+- Delete object:
```
Comment (navarroaxel) [inline pages/common/s3cmd.md:30]:
```suggestion
- Delete a specific object:
```

File: pages/common/s3cmd.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- List Buckets/Folders/Objects:
+
+`s3cmd ls s3://{{bucket}}`
+
+- Make Bucket/Folder:
+
+`s3cmd mb s3://{{bucket}}`
+
+- Get file from Bucket:
```
Comment (navarroaxel) [inline pages/common/s3cmd.md:18]:
```suggestion
- Download a specific from a bucket:
```

File: pages/common/s3cmd.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`s3cmd get s3://{{bucket}}/{{file}} {{local_file}}`
```
Comment (navarroaxel) [inline pages/common/s3cmd.md:20]:
```suggestion
`s3cmd get s3://{{bucket_name}}/{{path/to/file}} {{path/to/local_file}}`
```

File: pages/common/s3cmd.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# s3cmd
+
+> Command line tool and client for uploading, retrieveing and managing data in S3 compatible object storage.
+> More information: <https://s3tools.org/s3cmd>.
+
+- Initial configuration:
+
+`s3cmd --configure`
+
+- List Buckets/Folders/Objects:
```
Comment (navarroaxel) [inline pages/common/s3cmd.md:10]: How this can list buckets?

Comment (bgbak) [reply pages/common/s3cmd.md:10]: `s3cmd ls` by itself will list buckets.
`s3cmd ls s3://{{bucket|path}}`will list contents of that bucket/folder

The s3cmd manpage states:
```
s3cmd ls [s3://BUCKET[/PREFIX]] 
  List objects or buckets
```

I'm not sure how to format the s3 path parameter as optional.

Comment (navarroaxel) [reply pages/common/s3cmd.md:10]: your explanation is not clear in this example. you are talking about `ls` without arguments and a specific bucket URI.

File: pages/common/s3cmd.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Invoke configuration/reconfiguration tool:
+
+`s3cmd --configure`
+
+- List Buckets/Folders/Objects:
+
+`s3cmd ls s3://{{bucket|path/to/file}}`
+
+- Make Bucket/Folder:
```
Comment (einverne) [inline pages/common/s3cmd.md:14]: Do you mean `create` a Bucket here?

**Review by einverne — APPROVED** [review]
Looks good.

File: pages/common/s3cmd.md (hunk 6 above)
Comment (bgbak) [reply pages/common/s3cmd.md:14]: Yes, "create" would be a better work. Thanks :)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**